ios/README.md

```diff
@@ -0,0 +1,86 @@
+
+        forceios
+        Usage:
+        forceios create
+            --apptype=<Application Type> (native, native_swift, react_native, hybrid_local, hybrid_remote)
+            --appname=<Application Name>
+            --packagename=<App Package Identifier> (com.mycompany.myapp)
+            --organization=<Organization Name> (Your company\'s/organization\'s name)
+            --startpage=<App Start Page> (The start page of your remote app. Only required for hybrid_remote)
+            [--outputdir=<Output directory> (Leave empty for current directory)]
+
+        OR
+
+        forceios createWithTemplate
+            --templaterepouri=<Template repo URI> (e.g. https://github.com/forcedotcom/SmartSyncExplorerReactNative)
+            --appname=<Application Name>
+            --packagename=<App Package Identifier> (com.mycompany.myapp)
+            --organization=<Organization Name> (Your company\'s/organization\'s name)
+            [--outputdir=<Output directory> (Leave empty for current directory)]
+
+        OR
+
+        forceios version
+
+        OR
+
+        forceios
+
+**Note:** You can specify any or all of the arguments as command line options as specified in the usage.  If you run `forceios create` with missing arguments, it prompts you for each missing option interactively.
+
+Once the creation script completes, you'll have a fully functioning basic application of the type you specified.  The new application is an Xcode project that you can peruse, run, and debug.
+
+### forceios create options
+
+**App Type:** The type of application you wish to develop:
+
+- **native** — A fully native iOS application
+- **hybrid\_remote** — A hybrid application, based on the [Cordova](http://cordova.apache.org/) framework, that runs in a native container.  The app contents live in the cloud as a [Visualforce](http://wiki.developerforce.com/page/An_Introduction_to_Visualforce) application
+- **hybrid\_local** — A hybrid application, based on the Cordova framework, that runs in a native container.  The app contents are developed locally in the Xcode project, and are deployed to the device itself when the app is built
+
+**App Name:** The name of your application
+
+**Company ID:** An identifier for your company, similar to a Java package (e.g. `com.acme.MobileApps`).  This concatenates with the app name to form the unique identifier for your app in the App Store.
+
+**Organization:** The name of your company or organization.  For example, `Acme Widgets, Inc.`
+
+**Start Page:** \( *Required for hybrid\_remote apps only* \) The starting page of your application on salesforce.com.  This is the entry point of your remote application, though it's only the path, not the server portion of the URL.  For instance, `/apex/MyVisualforceStartPage`.
+
+**Output Directory:** \( *optional* \) The directory where you want your app to be created.  If not specified, it will be created in your current working directory.
```

Comment:
We should copy those two lines into the forcedroid readme - because it behaves that way now